### PR TITLE
Added security-header example

### PR DESCRIPTION
**Pull request by pascalandy**

### CODE REVIEW

File: security-header/Caddyfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+header / {
+    Strict-Transport-Security "max-age=31536000; includeSubDomains; preload"
```
Comment (mholt) [inline security-header/Caddyfile]: Should probably mention on the readme a warning before site owners unknowingly shoot themselves in the foot (those who copy and paste without reading)...

File: security-header/README.md [hunk 2]
```diff
@@ -4,7 +4,22 @@ Get an **A+** on [security header](https://securityheaders.com/) out of the box.
 
 Before I was using CloudFlare worker (5$ per month) to get the same result. I asked myself, how does the ‘real server’ are configured to manage the scurity-header? This is how.
 
-**screenshot**:
+# Warning
+
+Test those default values locally as they could prevent your website functioning normally.
+
+- Content-Security-Policy	Content Security Policy is an effective measure to protect your site from XSS attacks. By whitelisting sources of approved content, you can prevent the browser from loading malicious assets. Analyse this policy in more detail. You can sign up for a free account on Report URI to collect reports about problems on your site.
```
Comment (mholt) [inline security-header/README.md]: Could we format these bullet items a little nicer? The extra spacing doesn't do anything in reality and it all runs together. Try bolding instead.

File: security-header/README.md [hunk 3]
```diff
@@ -4,7 +4,22 @@ Get an **A+** on [security header](https://securityheaders.com/) out of the box.
...
+- Feature-Policy	Feature Policy is a new header that allows a site to control which features and APIs can be used in the browser.
+- Referrer-Policy	Referrer Policy is a new header that allows a site to control how much information the browser includes with navigations away from a document and should be set by all sites.
+- Strict-Transport-Security	HTTP Strict Transport Security is an excellent feature to support on your site and strengthens your implementation of TLS by getting the User Agent to enforce the use of HTTPS.
```
Comment (mholt) [inline security-header/README.md]: .... buuuuut they risk bricking their site if they ever don't use HTTPS -- be sure to mention the actual warnings! Don't just describe the feature.

File: security-header/README.md [hunk 4]
```diff
@@ -4,7 +4,22 @@ Get an **A+** on [security header](https://securityheaders.com/) out of the box.
...
+- X-Content-Type-Options	X-Content-Type-Options stops a browser from trying to MIME-sniff the content type and forces it to stick with the declared content-type. The only valid value for this header is "X-Content-Type-Options: nosniff".
+- X-Frame-Options	X-Frame-Options tells the browser whether you want to allow your site to be framed or not. By preventing a browser from framing your site you can defend against attacks like clickjacking.
+- X-Xss-Protection	X-XSS-Protection sets the configuration for the cross-site scripting filters built into most browsers. The best configuration is "X-XSS-Protection: 1; mode=block".
+- Expect-CT	Expect-CT allows a site to determine if they are ready for the upcoming Chrome requirements and/or enforce their CT policy.
+- Server	Server value has been changed. Typically you will see values like "Microsoft-IIS/8.0" or "nginx 1.7.2".
```
Comment (mholt) [inline security-header/README.md]: Is this a warning?

**Comment by pascalandy** [comment]
@mholt done :)

**Comment by mholt** [comment]
That should work -- thanks!

**Comment by pascalandy** [comment]
I'm the one that has to thank you!
Cheers!

**Pull request merged**